packages/@react-aria/focus/src/FocusScope.tsx

```diff
@@ -184,6 +184,19 @@ function getScopeRoot(scope: HTMLElement[]) {
 function useFocusContainment(scopeRef: RefObject<HTMLElement[]>, contain: boolean) {
   let focusedNode = useRef<HTMLElement>();
 
+  // Loop through the scopeRef array, and check if scopeRef contains
+  // a FocusScope element (determined by two span children). If it does,
+  // pop it.
+  scopeRef.current.forEach(element => {
+    // How do I check for a FocusScope element? Maybe check if it
+    // has two spans. This might not be the best solution for future
+    // development
+    var arr = [...element.children]
+    arr.forEach((item, index, object) => {
+      if (item.nodeName == "span") object.splice(index, 1);
```

Comment:
Checking by just `span` tag can lead to some very weird breaking changes for users of this component - I believe it wouldn't be unique enough to check by just `span` tag name. Maybe instead we could add some custom `data-` attribute that would indicate clearly it's a focus scope? Something like `<span data-focus-scope-start hidden />` and `<span data-focus-scope-end hidden />`